.editorconfig

```diff
@@ -0,0 +1,207 @@
+# Remove the line below if you want to inherit .editorconfig settings from higher directories
+root = true
+
+# C# files
+[*.cs]
+
+#### Core EditorConfig Options ####
+
+# Indentation and spacing
+indent_size = 4
+indent_style = space
+tab_width = 4
+trim_trailing_whitespace = true
+
+# New line preferences
+end_of_line = crlf
+insert_final_newline = true
+
+#### .NET Coding Conventions ####
+
+# Organize usings
+dotnet_separate_import_directive_groups = false
+dotnet_sort_system_directives_first = false
```

Comment:
I think `dotnet_sort_system_directives_first` should be `true` to better reflect the current project style.

@GPSnoopy do you confirm?

Reply:
pushed with `dotnet_sort_system_directives_first = true`
let me know how to proceed 🕺